src/main.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import discord
+from discord.ext import commands
+
+bot_prefix = os.getenv("PREFIX") or "p."
+
+bot = commands.Bot(
+    command_prefix=commands.when_mentioned_or(bot_prefix),  # type: ignore
```

Comment:
prefix only for sync command

Reply:
Shouldn't we sync on bot startup?

https://github.com/Rapptz/discord.py/blob/86dd29a8e8ceeb9db4086a12b73a79b07ef04aef/examples/app_commands/basic.py#L27

Reply:
There are chance that it fail to sync correctly and it would be annoying, and many bot maker would generally agree that we should only sync after command

Reply:
Alright then, we could do that